force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -83,7 +109,12 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the number is even, False otherwise.
      */
     public static Boolean isEven(Integer num) {
-        return null; // Replace null with the variable you used to store the result
+        if (Math.mod(num,2) == 0) {
+            return true;
+        }
+        else {
+            return false;
+        }
```

Comment:
Something to think about when you are coding  is  what does my expression evaluate too. In this case your method returns a boolean. And   your expression Math.mod... will evaluate to a boolean. So  you could do this. 

```
	public static Boolean isEven(Integer num) {
		return Math.mod(num, 2) == 0;
	}
```